v2.1/reference/calicoctl/commands/node/run.md

```diff
@@ -115,6 +115,15 @@ Description:
   Calico networking and network policy on your compute host.
 ```
 
+### Kubernetes as the datastore
+
+If you are using Kubernetes as the Calico datastore, BGP routing is *currently* 
```

Comment:
We should standardize on what we call this.

Should be "Kubernetes as the datastore"? "Kubernetes datastore driver"? "Kubernetes API as the datastore"?  "Kubernetes API datastore driver"? 

I'm not sure which is most clear. I think "Kubernetes API as the datastore"?

minor nit you could choose to ignore: I prefer wording it as "When Calico is configured to use the Kubernetes..." rather than "When _you_ are using".